boa/src/builtins/number/mod.rs

```diff
@@ -417,7 +417,23 @@ impl Number {
         make_builtin_fn(Self::to_string, "toString", &prototype, 1);
         make_builtin_fn(Self::value_of, "valueOf", &prototype, 0);
 
-        make_constructor_fn(Self::make_number, global, prototype)
+        let number = make_constructor_fn(Self::make_number, global, prototype);
+
+        // Constants from:
+        // Draft ECMA-262 / May 22, 2020 ECMAScript® 2021 Language Specification
+        // Section 20.1.2 Properties of the Number Constructor
+        number.set_field("EPSILON", Value::from(2.220_446_049_250_313_080_847_263_336_181_6E-16_f64));
+        number.set_field("MAX_SAFE_INTEGER", Value::from(9_007_199_254_740_991_f64));
+        number.set_field("MIN_SAFE_INTEGER", Value::from(-9_007_199_254_740_991_f64));
+        number.set_field("MAX_VALUE", Value::from(1.797_693_134_862_315_7E+308_f64));
+        number.set_field("MIN_VALUE", Value::from(5E-324_f64));
```

Comment:
```suggestion
        number.set_field("MAX_VALUE", Value::from(std::f64::MAX));
        number.set_field("MIN_VALUE", Value::from(std::f64::MIN));
```
We can use `std::f64::MAX` and `std::f64::MIN` here.